authzed/api/v0/materialize/watchpermissions.proto

```diff
@@ -0,0 +1,87 @@
+syntax = "proto3";
+package authzed.api.materialize.v0;
+
+import "authzed/api/v1/core.proto";
+import "google/protobuf/struct.proto";
+import "validate/validate.proto";
+
+option go_package = "github.com/authzed/authzed-go/proto/authzed/api/v0/materialize";
+option java_package = "com.authzed.api.materialize.v0";
+option java_multiple_files = true;
+
+service WatchPermissionsService {
+  // WatchPermissions returns a stream of PermissionChange events for the given permissions.
+  //
+  // WatchPermissions is a long-running RPC, and will stream events until the client
+  // closes the connection or the server terminates the stream. The consumer is responsible of
+  // keeping track of the last seen revision and resuming the stream from that point in the event
+  // of disconnection or client-side restarts.
+  //
+  // The API does not offer a sharding mechanism and thus there should only be one consumer per target system.
+  // Implementing an active-active HA consumer setup over the same target system will require coordinating which
+  // revisions have been consumed in order to prevent transitioning to an inconsistent state.
+  //
+  // Usage of WatchPermissions requires it to be explicitly enabled on the service,
```

Comment:
Indicate that the permissions watched needs to be explicitly set on the service side